app/uk/gov/hmrc/exports/services/mapping/CachingMappingHelper.scala

```diff
@@ -33,19 +36,20 @@ class CachingMappingHelper {
       ),
       dangerousGoods = exportItem.dangerousGoodsCode.flatMap(_.dangerousGoodsCode).map(code => Seq(DangerousGoods(Some(code)))).getOrElse(Seq.empty)
     )
+  }
 
   private def getClassifications(
-    commodityCode: Option[String],
-    cusCode: Option[String],
-    nationalAdditionalCodes: Seq[String],
-    taricCodes: Seq[String]
-  ): Seq[Classification] =
+                                  commodityCode: Option[String],
+                                  cusCode: Option[String],
+                                  nationalAdditionalCodes: Seq[String],
+                                  taricCodes: Seq[String]
+                                ): Seq[Classification] =
```

Comment:
This part will fail scalafmt

Reply:
That must mean that you IDE is not set up to run scalafmt.
You can do it from the command line using `sbt scalafmt test:scalafmt` - that will fix-up and files that have problems.